source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -91,7 +91,64 @@ After you’ve made an authorisation request for authentication and identity, yo
 
 You can only request user attributes to be returned in the `/userinfo` response. You cannot configure the claims returned in the [ID token][integrate.understand-id-token].
 
-### Replace your example message’s placeholder values
+### Secure your authorisation request parameters with JWT 
+
+You can use JWT-secured OAuth 2.0 (JAR) to encode your request parameters and protect against unwanted tampering with your request. 
```

Comment:
```suggestion
You can use a JWT-secured OAuth 2.0 authorisation request (JAR) with encoded parameters to protect your request from attacks and hackers. 
```
Wondering if this works so we can use the full JAR expanded acronym? Although I note that JAR can also mean 'Java archive', which is what I know the acronym for - and we used it in the Verify docs. Are we sure we're not going to need to refer to that other meaning in this docs set? I only saw it in the [Verify legacy docs](https://github.com/search?q=repo%3Aalphagov%2Fverify-tech-docs%20JAR&type=code), so hopefully not!

And I thought 'attacks and hackers' might be more direct about the potential consequences if users don't use the JAR method, but I know you will have more context on that :)

Reply:
@elenaschwan Just checking about the first bit of my comment ^!

Reply:
Apologies, I missed your comment, but I am just following up with Paul on this.

Reply:
this looks good to me! I'm happy with "protect your request from attacks and hackers"